### PR TITLE
feat: Add serverless vote information augmentation function

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds a very simple serverless function that returns information about the current staking APY, the current number of active requests and the current vote phase to be used within the new UMA.xyz website.

To use this serverless function you hit the URL of the deployment `/get-voting-info` and it will return a structured object like this: 

```
{
	"apy": "48",
	"activeRequests": "1",
	"phase": "Commit"
}
```

Note this function requires two envs to be added:

```
CHAIN_ID = 1 // (or 5 for goerli)
NODE_URLS ={"1":"https://mainnet.infura.io/v3/xxx","5":"https://goerli.infura.io/v3/xxx"}
```

This way you can set one ENV for all chains and then just switch on the `CHAIN_ID` to move between the testnet/prod version of the function.

### CODE REVIEW

**Comment by Tulun** [comment]
@chrismaree can you add documentation to any environment variables you need?

**Comment by chrismaree** [comment]
> @chrismaree can you add documentation to any environment variables you need?

good callout, done in the PR description!

**Review by ryanwolhuter — APPROVED** [review]
code lgtm but I haven't tested it out in the actual app yet

File: pages/api/_common.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return JSON.parse(process.env.NODE_URLS);
+}
+
+export async function constructContract(contractName: any) {
+  if (!process.env.CHAIN_ID) throw Error("CHAIN_ID env variable not set!");
+  const chainId = Number(process.env.CHAIN_ID);
+
+  // The latest release on goerli is not up to date. This is a temp work around so this will work on the testnet.
+  const contractAddress =
+    chainId == 5 ? "0xb3AED6A86B43De70dfe814C5ac549e722CF635D2" : await getAddress(contractName, chainId);
```
Comment (Tulun) [inline pages/api/_common.ts:26]: Should this address be an .env? or will this eventually get updated in protocol so we can just delete it?

**Review by Tulun — APPROVED** [review]
LGTM, integration can be done in a different PR.

**Pull request merged**